spec/support/rbac_helper.rb

```diff
@@ -0,0 +1,31 @@
+module Spec
+  module Support
+    module RBACHelper
```

Comment:
You don't need the modules around, just put it into the folder where other shared examples live

Reply:
👍